packages/spor-react/src/theme/components/breadcrumb.ts

```diff
@@ -21,27 +19,27 @@ const baseStyleLink = defineStyle({
     paddingX: 0.5,
     borderRadius: "xs",
     _hover: {
-      backgroundColor: "coralGreen",
+      backgroundColor: mode("seaMist", "pine")(props),
     },
     _focusVisible: {
       boxShadow: getBoxShadowString({
-        borderColor: "greenHaze",
+        borderColor: mode("greenHaze", "azure")(props),
```

Comment:
Denne gir fortsatt ikke riktig fokus på firefox. Denne kodesnytten jeg forslo tidligere, som bytter ut `_focusVisible` og erstatter med kodeblokken under tar hånd om saken.
   ```typescript
    ...focusVisible({
      focus: {
        boxShadow: getBoxShadowString({
          borderColor: mode("greenHaze", "azure")(props),
          borderWidth: 2,
        }),
      },
      notFocus: {
        notFocus: { boxShadow: "none" },
      }
    }),
    ```
    
    For ordensskyld så er disse skjermbildene med og uten denne blokka:

![Screenshot 2023-10-03 at 09 30 45](https://github.com/nsbno/spor/assets/3692249/22597cea-da86-472d-aab8-58702976dd48)
![Screenshot 2023-10-03 at 09 32 19](https://github.com/nsbno/spor/assets/3692249/b7c777fb-b566-44d1-8a7f-920af8fe07d9)

Reply:
![Screenshot 2023-10-03 at 09 30 45](https://github.com/nsbno/spor/assets/3692249/0c1e601e-ac27-44b9-8c2b-a26a8bf7cd31)
![Screenshot 2023-10-03 at 09 32 19](https://github.com/nsbno/spor/assets/3692249/7719f4a8-c186-4643-acae-379407d40648)